src/cookbook/design/fonts.md

```diff
@@ -131,9 +133,9 @@ as part of the app's `theme`. The value provided to
 `fontFamily` must match the `family`
 name declared in the `pubspec.yaml`.
 
-<!-- skip -->
```

Comment:
Every time I see one of these disappear, it makes me happy. 😄

Reply:
😄